ui/src/lib/forms/childPostnatalCare.js

```diff
@@ -0,0 +1,314 @@
+import * as yup from 'yup';
+import childpostnataldata from '../../data/child_postnatal_care.json';
+
+const childPostnatalCareFields = {
+  
+ 'BABY': [
+    {
+      name: 'generalTimingOfVisit',
+      label: 'Timing of visit',
+      type: 'select',
+      validate: yup.string().required('Timing of visit is required'),
+      width: {
+        xs: 12,
+        sm: 12,
+        md: 12,
+        lg: 5,
+      },
+      options: Object.keys(childpostnataldata).map(key => ({
+        value: key,
+        label: key,
+      })),
+    },
+    {
+      name: 'nextVisit',
+      label: 'Date/Visit',
+      type: 'date',
+      validate: yup
+        .date()
+        .max(new Date(), 'Date of visit cannot be in the future')
+        .required('Next visit is required'),
+      width: {
+        xs: 12,
+        sm: 12,
+        md: 12,
+        lg: 5,
+      },
+    },
+  ],
+
+  'General Condition':
+  [
+    {
+      name: 'generalCondition',
+      label: 'General Condition',
+      type: 'radio',
+      validate: yup.string(),
+      width: {
+        xs: 12,
+        sm: 12,
+        md: 12,
+        lg: 12,
+      },
+      options: [
+        { label: 'Well', value: 'Well' },
+        { label: 'Unwell', value: 'Unwell' },
+      ],
+    },
+    {
+      name: 'generalConditionSpecify',
+      label: 'If Unwell, please specify',
+      type: 'text',
+      validate: yup.string(),
+      width: {
+        xs: 12,
+        sm: 12,
+        md: 12,
```

Comment:
Change width for "md" to 8 and "lg" to 6 for all fields of this kind i.e "specify" fields